Readme.md

```diff
@@ -6,7 +6,9 @@ All variables can be changed via the ingame mod settings (needs game relaunch).
 
 ## Default Changes to Base Game
 
-* Charging Speed per Robot `1MW` -> `10MW`
+* Charging Speed per Robot `1MW` -> `4MW`

```

Comment:
You change the value in the settings.lua to `1MW`.  Im fine with this as it is the game default. I would just note this here with something like this:
```suggestion
* Charging Speed per Robot stays at default `1MW`
```

Reply:
Yes. I made this change and did not reflect it in the documentation. Your correction is totally fine.